### PR TITLE
feat: added social media component

### LINKED ISSUE #44: Component: Footer / SocialMedia
Social media component.

### PR DESCRIPTION
**Pull request by TeddTech**
resolves #44

### CODE REVIEW

File: assets/config.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ const Config = {
 	wpDomain: "https://wecount.inclusivedesign.ca",
 	loadDbName: "starter_wp",
 	apiBase: "/wp-json/wp/v2/",
-	contactEmail: "wecount@inclusivedesign.ca"
+	contactEmail: "wecount@inclusivedesign.ca",
+	socialMedia: [{ title: "Twitter", href: "https://twitter.com/wecount" }]
```
Comment (greatislander) [inline assets/config.js]: @TeddTech I'd actually leave this as an empty array.

Comment (TeddTech) [reply assets/config.js]: Like an empty string? `""`

Comment (greatislander) [reply assets/config.js]: No, an empty array: `[]`. We don't have any social media so we should not provide any.

Comment (TeddTech) [reply assets/config.js]: Oh okay. Thought you were talking about `contactEmail`

Comment (TeddTech) [reply assets/config.js]: It is completed now

**Pull request merged**